go/test/endtoend/vtgate/queries/vexplain/vexplain_test.go

```diff
@@ -53,6 +53,7 @@ func start(t *testing.T) (*mysql.Conn, func()) {
 }
 
 func TestVtGateVExplain(t *testing.T) {
+	t.Skip("v22 changes the output of vexplain queries because of binary bind vars which breaks this test.")
```

Comment:
This is fine too, but we can also only skip for v22 version of vtgate.

Reply:
I plan to clean this up / improve this once I have https://github.com/vitessio/vitess/pull/16988 landed on main.